scanner/updater/export.go

```diff
@@ -59,34 +59,32 @@ func Export(ctx context.Context, outputDir string) error {
 
 	var opts [][]updates.ManagerOption
 
-	// Manual CVEs.
-	manualSet, err := manual.UpdaterSet(ctx, nil)
-	if err != nil {
-		return err
-	}
-	outOfTree := append([]driver.Updater{}, manualSet.Updaters()...)
-	opts = append(opts, []updates.ManagerOption{updates.WithOutOfTree(outOfTree)})
-
-	// RHEL custom: Forked from ClairCore.
-	fac, err := rhel.NewFactory(ctx, rhel.DefaultManifest)
-	if err != nil {
-		return err
-	}
+	// NVD enricher.
 	opts = append(opts, []updates.ManagerOption{
-		updates.WithFactories(map[string]driver.UpdaterSetFactory{"rhel-custom": fac})})
+		updates.WithFactories(map[string]driver.UpdaterSetFactory{
+			"nvd": nvd.NewFactory(),
+		}),
+		updates.WithConfigs(map[string]driver.ConfigUnmarshaler{
+			"nvd": func(i interface{}) error {
+				cfg, ok := i.(*nvd.Config)
+				if !ok {
+					return errors.New("internal error: config assertion failed")
+				}
+				ci := os.Getenv("STACKROX_NVD_API_CALL_INTERVAL")
+				if ci != "" {
+					cfg.CallInterval = &ci
+				}
+				key := os.Getenv("STACKROX_NVD_API_KEY")
+				if key != "" {
+					cfg.APIKey = &key
+				}
+				return nil
+			},
+		}),
+	})
 
 	// ClairCore updaters.
-	for _, uSet := range [][]string{
```

Comment:
TBH, I don't quite understand this change. Do we not need the other vulnerability data to be generated? Or maybe I am missing something.

Reply:
This was a manual testing step that was pushed by mistake. Thanks for letting me know!